frontend/src/utils/constants.js

```diff
@@ -114,7 +114,7 @@ export const APPLICATION_ERROR_MESSAGES = Object.freeze({
   EXPENSE_AUTHORITY: 'Expense authority required',
   LICENCE_INFO: 'Licence information required',
   LICENCE_CONFIRMATION: 'Confirmation of licence information required',
-  DOCUMENT_UPLOAD: 'Document upload required',
+  DOCUMENT_UPLOAD: 'A document upload for Financial Statement and Balance Sheet are required',
```

Comment:
we should remove 'are' to keep the error message consistent

Reply:
Fixed.